pkgs/by-name/di/disnix-web-service/package.nix

```diff
@@ -27,8 +36,8 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   env = {
-    PREFIX = "\${env.out}";
-    AXIS2_LIB = "${axis2}/lib";
+    PREFIX = placeholder "out";
+    AXIS2_LIB = "${axis2}/webapps/axis2/WEB-INF/lib";
```

Comment:
Is this supposed to be part of this PR? Did this just not work before? Is this even used? The sed below modifies the same variable but uses the previous value?

Reply:
It broke due to an axis2 update, I think, and I forgot the sed.